src/main/java/in/org/projecteka/hiu/dataprocessor/HealthDataProcessor.java

```diff
@@ -287,15 +289,18 @@ private ProcessedEntry processEntryContent(DataContext context,
                     }
                 });
                 result.setEncoded(parser.encodeResourceToString(bundle));
+                result.setUniqueResourceId(bundleContext.getBundleUniqueId());
+                result.setDocumentType(bundleContext.getDocumentType());
                 result.addTrackedResources(bundleContext.getTrackedResources(), bundleContext.getBundleDate());
                 return result;
             } catch (Exception e) {
                 logger.error("Could not process bundle {exception}", e);
-                result.getErrors().add(String.format("Could not process bundle with id: %s, error-message: %s",
+                result.addError(String.format("Could not process bundle with id: %s, error-message: %s",
                         bundle.getId(), e.getMessage()));
                 return result;
             }
         }).orElseGet(() -> {
+            ProcessedEntry result = new ProcessedEntry();
```

Comment:
Are we using it anywhere? It doesn't seem to be used

Reply:
It used to be outside, have pushed it inside the block. The result.errors are used in the caller function

Reply:
I see it only in this scope so I wonder how can it be used outside of this scope.